README.md

```diff
@@ -381,10 +381,21 @@ collection.prevPage({ update: true, remove: false });
 
 ## Development
 
+Create a development environment with Docker:
+
+```
+docker build -t marionette.carpenter:latest .
+docker run --rm -it -w $(pwd) -v $(pwd):$(pwd) marionette.carpenter:latest /bin/bash
+npm install -g bower && \
```

Comment:
This should be moved into the docker container; but there were some issues with mise that I didn't have the cycles to investigate further